sklearnex/tests/test_patching.py

```diff
@@ -41,50 +44,110 @@ def run_parse(mas, result):
             result[ind] = get_branch(temp)
             temp.clear()
         else:
             temp.append(mas[i])
 
 
 def get_result_log():
-    os.environ['SKLEARNEX_VERBOSE'] = 'INFO'
+    os.environ["SKLEARNEX_VERBOSE"] = "INFO"
     absolute_path = str(pathlib.Path(__file__).parent.absolute())
     try:
         process = subprocess.check_output(
-            [
-                sys.executable,
-                absolute_path + '/utils/_launch_algorithms.py'
-            ]
+            [sys.executable, absolute_path + "/utils/_launch_algorithms.py"]
         )
     except subprocess.CalledProcessError as e:
         print(e)
         exit(1)
     mas = []
     result = {}
-    for i in process.decode().split('\n'):
-        if i.startswith('SKLEARNEX WARNING'):
+    for i in process.decode().split("\n"):
+        if i.startswith("SKLEARNEX WARNING"):
             continue
-        if not i.startswith('SKLEARNEX INFO') and len(mas) != 0:
+        if not i.startswith("SKLEARNEX INFO") and len(mas) != 0:
             run_parse(mas, result)
             mas.clear()
             mas.append(i.strip())
         else:
             mas.append(i.strip())
-    del os.environ['SKLEARNEX_VERBOSE']
+    del os.environ["SKLEARNEX_VERBOSE"]
     return result
 
 
 result_log = get_result_log()
 
 
-@pytest.mark.parametrize('configuration', result_log)
+@pytest.mark.parametrize("configuration", result_log)
 def test_patching(configuration):
-    if 'OPT' in result_log[configuration]:
+    if "OPT" in result_log[configuration]:
         return
     for skip in TO_SKIP:
         if re.search(skip, configuration) is not None:
             pytest.skip("SKIPPED", allow_module_level=False)
-    raise ValueError('Test patching failed: ' + configuration)
+    raise ValueError("Test patching failed: " + configuration)
+
+
+def _load_all_models(patched):
+    if patched:
+        patch_sklearn()
+
+    from sklearn.cluster import DBSCAN, KMeans
+    from sklearn.decomposition import PCA
+    from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+    from sklearn.linear_model import (
+        ElasticNet,
+        Lasso,
+        LinearRegression,
+        LogisticRegression,
+        Ridge,
+    )
+    from sklearn.manifold import TSNE
+    from sklearn.neighbors import (
+        KNeighborsClassifier,
+        KNeighborsRegressor,
+        LocalOutlierFactor,
+        NearestNeighbors,
+    )
+    from sklearn.svm import SVC, SVR, NuSVC, NuSVR
+
+    models = [
+        DBSCAN(),
+        ElasticNet(),
+        KMeans(),
+        KNeighborsClassifier(),
+        KNeighborsRegressor(),
+        Lasso(),
+        LinearRegression(),
+        LocalOutlierFactor(),
+        LogisticRegression(),
+        NearestNeighbors(),
+        NuSVC(),
+        NuSVR(),
+        PCA(),
+        RandomForestClassifier(),
+        RandomForestRegressor(),
+        Ridge(),
+        SVC(),
+        SVR(),
+        TSNE(),
+    ]
```

Comment:
It is better to get estimators from sklearnex.dispatcher::get_patch_map without need to update list of estimators manually. Entry of mapping contains sklearn submodule and estimator name which you can get:
```
>>> mapping['pca']
[[(<module 'sklearn.decomposition' from 'miniconda3/envs/build_py39/lib/python3.9/site-packages/sklearn/decomposition/__init__.py'>, 'PCA', <class 'daal4py.sklearn.decomposition._pca.PCA'>), None]]
```

Reply:
Trying to add it, but I really don't get the structure of this thing. Why the heck is it a list of lists of tuples. There is only a single tuple. Why not flatten this?

Reply:
It does not produce the correct instances

```python
    models = [
        patch_infos[0][0][2]()
        for key, patch_infos in get_patch_map().items()
        if key
        not in (
            "distances",
            "logistic",
            "train_test_split",
            "fin_check",
            "roc_auc_score",
            "set_config",
            "get_config",
            "config_context",
        )
    ]
```

![image](https://user-images.githubusercontent.com/9201869/230058439-dc62750f-c2f4-46f8-af73-9bc503159590.png)

Reply:
Okay, I think I see now what you mean. Will try importlib